### PR TITLE
Simplify models

### PR DESCRIPTION
**Pull request by Muennighoff**
This is the branch the Arena is currently running on. Made it such that task types can also be in the kwargs as here https://github.com/embeddings-benchmark/arena/blob/b5bffc8137cec0c55f0bc472a3255867388f0526/models.py#L286

### CODE REVIEW

File: mteb/models/gte_models.py [hunk 1]
```diff
@@ -1,3 +1,44 @@
...
+  File "<stdin>", line 1, in <module>
+TypeError: SentenceTransformer.encode() got an unexpected keyword argument 'instruction'
+>>> query_embeddings_mteb = model_mteb.encode(queries, instruction="Given a web search query, retrieve relevant passages that answer the query")
+>>> document_embeddings_mteb = model_mteb.encode(documents)
+>>> scores_mteb = (query_embeddings @ document_embeddings.T) * 100
+>>> print(scores_mteb.tolist())
+[[70.39706420898438, 3.4318461418151855], [4.516170978546143, 81.91815948486328]]
+>>>
+
+"""
```
Comment (KennethEnevoldsen) [inline mteb/models/gte_models.py:40]: I would but it down in if __name__ == __main__

File: mteb/models/voyage_models.py [hunk 2]
```diff
@@ -199,8 +199,8 @@ def _batched_encode(
     embed_dim=1024,
     open_source=False,
 )
-
-voyage_multilingual_2 = ModelMeta(  # reference: https://blog.voyageai.com/2024/06/10/voyage-multilingual-2-multilingual-embedding-model/"
+# see https://blog.voyageai.com/2024/06/10/voyage-multilingual-2-multilingual-embedding-model/"
```
Comment (KennethEnevoldsen) [inline mteb/models/voyage_models.py:202]: There is a reference key in ModelMeta would move it there

File: mteb/models/google_models.py [hunk 3]
```diff
@@ -37,16 +37,18 @@ def _embed(
...
         else:
-            inputs = [TextEmbeddingInput(text, task_type=task_type) for text in texts]
+            inputs = [
+                TextEmbeddingInput(text, task_type=google_task_type) for text in texts
+            ]
         kwargs = dict(output_dimensionality=dimensionality) if dimensionality else {}
         try:
             embeddings = model.get_embeddings(inputs, **kwargs)
-        # Except the very rare google.api_core.exceptions.InternalServerError
+        # # Except the very rare google.api_core.exceptions.InternalServerError
```
Comment (KennethEnevoldsen) [inline mteb/models/google_models.py:51]:
```suggestion
        # Except the very rare google.api_core.exceptions.InternalServerError
```

**Review by KennethEnevoldsen — APPROVED** [review]
Only a few minor changes - have we ensure that it still works on a candidate mteb task?

File: mteb/models/voyage_models.py (hunk 2 above)
Comment (Muennighoff) [reply mteb/models/voyage_models.py:202]: Hm I don't see one: https://github.com/embeddings-benchmark/mteb/blob/700a41d274189f3214ae7551d664f76192b0f49c/mteb/model_meta.py#L63 - which one is it?

**Pull request merged**